pkg/ccl/changefeedccl/changefeedbase/options.go

```diff
@@ -160,3 +163,19 @@ var ChangefeedOptionExpectValues = map[string]sql.KVStringOptValidate{
 	OptWebhookClientTimeout:     sql.KVStringOptRequireValue,
 	OptOnError:                  sql.KVStringOptRequireValue,
 }
+
+// ChangefeedOptionsAllSinks is options common to all sinks
+var ChangefeedOptionsAllSinks = importccl.MakeStringSet(OptCursor, OptEnvelope,
```

Comment:
I think we know we're dealing w/ changefeed options.
I would just shorten this var, as well as ones below:
CommonOptions, KafkaValidOptions, ChangefeedValidOptions.